src/App.tsx

```diff
@@ -17,18 +83,31 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                filterOption={filterOption}
+                onFilterSelect={onFilterSelect}
+                onFilterChange={onFilterChange}
```

Comment:
[checklist](https://github.com/mate-academy/react_dynamic-list-of-todos/blob/master/checklist.md): follow [these](https://medium.com/javascript-in-plain-english/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364) naming conventions